gix-merge/tests/fixtures/tree-baseline.sh

```diff
@@ -475,8 +475,8 @@ git init same-rename-different-mode
 
   git checkout A
   write_lines 1 2 3 4 5 >a/x.f
-  chmod +x a/x.f
-  chmod +x a/w
+  chmod +x a/x.f a/w
```

Comment:
Loving this - thanks to the concatenation of the initial `chmod` call there is a 'free' call to improve compatibility.